plugins/optimization-detective/site-health.php

```diff
@@ -127,6 +127,7 @@ function od_compose_site_health_result( $response ): array {
 		$message = wp_remote_retrieve_response_message( $response );
 		$body    = wp_remote_retrieve_body( $response );
 		$data    = json_decode( $body, true );
+		$header  = wp_remote_retrieve_header( $response, 'content-type' );
```

Comment:
Technically this can return an array of the response has more than one header of the same name. So this should probably do:

```suggestion
		$header  = wp_remote_retrieve_header( $response, 'content-type' );
		if ( is_array( $header ) ) {
			$header = array_pop( $header );
		}
```